test/format/checkpoint.c

```diff
@@ -142,7 +140,18 @@ checkpoint(void *arg)
         else
             trace_msg(session, "Checkpoint #%u start (%s)", ++counter, ckpt_config);
 
-        testutil_check(session->checkpoint(session, ckpt_config));
+        ret = session->checkpoint(session, ckpt_config);
+        /*
+         * Because of the concurrent activity of the sweep server, it is possible to get EBUSY when
+         * we are trying to remove an existing checkpoint as the sweep server may be interacting
+         * with a dhandle associated with the checkpoint being removed. There is a specific scenario
+         * where we remove an existing checkpoint if we create one with the same name.
+         */
+        if (ret == EBUSY)
+            testutil_assert(!WT_PREFIX_MATCH(ckpt_vrfy_name, WT_CHECKPOINT) ||
+              strncmp(ckpt_vrfy_name, "drop=(all)", strlen(ckpt_vrfy_name)));
```

Comment:
I think it would be easier to read and follow if you used a boolean above like `ebusy_ok` that was set to true in the named checkpoint and drop=all cases and false otherwise/top of the loop.